content/livesync/how-livesync-works.textile

```diff
@@ -0,0 +1,95 @@
+
+h3(#customer-support). Customer support applications
+
+Similarly to "CRM systems":#crm-systems above, LiveSync would assist on individual entities within the customer support application, rendering the details and updating when a change has been persisted to the database by another person or part of the system.
+
+h3(#task-management). Productivity applications / Task management apps
+
+Similarly to "CRM systems":#crm-systems above, LiveSync would assist on a specific task within a task management application, rendering the details and updating when a change has been persisted to the database by another person or part of the system.
+
+h3(#online-auction). Online auction
+
+An online auction application is the ability to list products on the platform and have bidders make bids on this product where at the end of the given time, the winner is the person with the highest bid.
+
+LiveSync can make it easier for bidders to view a single auction in realtime with changes in price reflected in every frontend clients view. Each auction item can be represented as a single data Model, and optimistic updates can make it possible for a bidder to submit a bit and see this immediately whether it is accepted by the server as the highest bid, or rejected because another user's bid is higher. This functionality means that the database can resolve two competing bids and store the outcome, which is reflected transactionally into Ably via the outbox table. Ably events and the database are never out of sync.
+
+h3(#collaborative). Collaborative form editing and building
+
+In collaborative form editing applications, LiveSync integrates seamlessly with tools like "Spaces":/products/spaces. Individual form elements, such as survey questions or presentation slides, can be locked for editing by users, with updates propagated to all users via channel events. Optimistic updates can be utilized for real-time validation during editing. It's crucial to use locking in such scenarios, as LiveSync follows a last-write-wins mechanism. Simultaneous editing of the same element by multiple users may lead to conflicts, resulting in the discarding of one user's updates.
+
+h3(#ecommerce). Inventory availability in ecommerce
+
+In a shopping application, LiveSync can be employed to manage inventory tracking for items. For instance, a single Model can represent available sizes and the corresponding item counts. When a user makes a purchase, the server processes the order and transactionally publishes an event with the updated size and inventory counts using the outbox. While optimistic updates could be applied to the inventory count, they may be unnecessary since users typically do not concern themselves with inventory availability post-purchase.
```

Comment:
>While optimistic updates could be applied to the inventory count, they may be unnecessary since users typically do not concern themselves with inventory availability post-purchase.

Fair, but not really necessary information to surface here imo - isn't the purpose of this section to highlight where livesync is strong?